code/modules/events/vent_clog.dm

```diff
@@ -74,6 +97,10 @@
 /datum/round_event/vent_clog/catastrophic
 	randomProbability = 30
 	reagentsAmount = 1250
```

Comment:
Your PR isn't addressing the amount of fluids in each vent.
I'd suggest 300/500/700